src/composables/Alchemy.ts

```diff
@@ -161,6 +161,146 @@ export const Vials: Vial[] = [
     base: 50,
     effect: "% Nothing. Absolutely nothing",
   },
+  {
+    name: "Fur Refresher",
+    roll: 75,
+    material: "Floof Ploof",
+    base: 2,
+    effect: "% higher Shiny Critter chance.",
```

Comment:
Just a heads-up: I am working on refactoring effects to use an enum instead of just a string. This way we can have some consistency with effects throughout the code, and it makes it easy to do quick calculations. This is fine for this PR though, but might lead to conflicts if #122 is merged first.